src/plumbing/progress.rs

```diff
@@ -14,10 +15,7 @@ enum Usage {
     /// We definitely want to implement this configuration value.
     Planned { note: Option<&'static str> },
     /// The configuration is already effective and used (at least) in the given module `name`.
```

Comment:
this comment is now wrong?

Reply:
Great catch! Indeed it was out of date, this should be fixed next time I push.